app/src/main/kotlin/com/bitwarden/authenticator/data/authenticator/manager/TotpCodeManager.kt

```diff
@@ -31,8 +31,14 @@ interface TotpCodeManager {
         const val PERIOD_PARAM = "period"
         const val SECRET_PARAM = "secret"
         const val ISSUER_PARAM = "issuer"
+
+        /**
+         * URI query parameter containing export data from Google Authenticator.
+         */
+        const val DATA_PARAM = "data"
```

Comment:
⛏️ IMO, If this is only needed on Google Authenticator, I'd prefix it with `GOOGLE_` or something that indicates that relation so one can guess without the comment what the param is about. Maybe even `GOOGLE_EXPORT_DATA_PARAM` to maintain the same pattern as below in `GOOGLE_EXPORT_PREFIX`.

Moreover, I'd move the Google specific constants to its own object so it's not mixed with the other general ones to keep the general implementation clean and then specific cases can be added elsewhere. So if then we need to add another authenticator's parameters/prefixes, we can do it cleanly in it's own object. Also with a specific object you save the `GOOGLE_` prefix.

Reply:
Noted, and I concur. We're going to reuse logic here in an upcoming PR, so I'll create a QR code parser abstraction in that one.